packages/js/plugins/ipfs/schema.graphql

```diff
@@ -24,10 +24,25 @@ type ResolveResult {
 }
 
 type Options {
```

Comment:
It'd be nice if we can have inside the ENV since all the call can share this vars, We also have option to override env at query time so I don't see any downside in using ENV here.

Reply:
I'd agree if we had the IPFS resolver and IPFS sdk plugins separated, but right now it gets confusing.
The Options here are used for the IPFS sdk part and is configurable per request (while using a single provider) while the QueryEnv down below is used for IPFS resolver part.

When we separate them we can have Env for both and configure them without the overlapping mess. 
TLDR: I'd hold off on this until we separate the plugins and leave this as is right now
Thoughts?

Reply:
That makes sense! And TBH it will only be this single release for which we will have this. In next release we need to refactor plugins anyway!